src/collectd.conf.pod

```diff
@@ -8083,24 +8083,25 @@ Global directives are:
 
 =over 4
 
-=item B<DNS_Cache_TTL> I<ttl>
+=item B<ResolveInterval> I<seconds>
 
-=item B<DNS_Random_Cache_TTL> I<ttl>
+=item B<ResolveJitter> I<seconds>
 
-When Collectd connects to a TSDB node, it will request the DNS. This can become
-a problem is the TSDN node is unavailable or badly configured because Collected
-will request DNS in order to reconnect for every metric, which can flood your DNS.
-So you can cache the last value for C<ttl> seconds (default: 600s e.g; 10 min).
+When I<collectd> connects to a TSDB node, it will request the DNS. This can
+become a problem if the TSDN node is unavailable or badly configured because
+collectd will request DNS in order to reconnect for every metric, which can
+flood your DNS. So you can cache the last value for I<ResolveInterval> seconds
+(default: B<600>, i.e. 10 minutes).
 
-You can also define a random ttl. This prevents all your Collectd servers to
-request the DNS at the same time when the connection fails. Default value is
-15 * the write_tsdb interval (or the global interval if write_tsdb interval is not
-defined).
+You can also define a jitter, a random interval to wait in addition to
```

Comment:
I would instead suggest calling this a "splay" in both config param and documentation, but just a minor nit.